ip: Do not skip serializing allow-extra-address: false

cathay4t opened the pull request:
The `nmstatectl fmt` will discard the `allow-extra-address` as it is
marked as skip serializing.

Normal query will hide this property as no backend provide such in
query result.

Unit test case included.

liangwen12year commented on rust/src/lib/ip.rs:
```diff
@@ -169,7 +168,7 @@ pub struct InterfaceIpv4 {
     /// Ignore when serializing.
     /// Deserialize from `allow-extra-address`
-    pub allow_extra_address: bool,
+    pub allow_extra_address: Option<bool>,
```
Include the `default` trait and dropping the `default_allow_extra_address()`, so the comment above need to be updated ?

cathay4t replied:
Comment updated.

cathay4t committed:
ip: Do not skip serializing allow-extra-address: false

The `nmstatectl fmt` will discard the `allow-extra-address` as it is
marked as skip serializing.

Normal query will hide this property as no backend provide such in
query result.

Unit test case included.

Signed-off-by: Gris Ge <fge@redhat.com>

cathay4t commented:
New patch uploaded. Please review.
